CONTRIBUTING.md

```diff
@@ -0,0 +1,249 @@
+* File a new issue if you run into problems!
+
+## Ground Rules
+
+The goal is to maintain a diverse community that's pleasant for everyone. Please be considerate and respectful of others by following our [code of conduct](https://github.com/ioos/.github/blob/main/CODE_OF_CONDUCT.md).
+
+Other items:
+
+* Each pull request should consist of a logical collection of changes. You can include multiple bug fixes in a single pull request, but they should be related.
+For unrelated changes, please submit multiple pull requests.
+* Do not commit changes to files that are irrelevant to your feature or bug fix
+(eg: .gitignore).
+* Be willing to accept criticism and work on improving your code; we don't want to break other users' code, so care must be taken not to introduce bugs.
+* Be aware that the pull request review process is not immediate, and is generally proportional to the size of the pull request.
+
+## Reporting a bug
+
+The easiest way to get involved is to report issues you encounter when using QARTOD or by requesting something you think is missing.
+
+* Head over to the project [issues](https://github.com/ioos/ioos_qc/issues) page.
+* Search to see if your issue already exists or has even been solved previously.
+* If you indeed have a new issue or request, click the "New Issue" button.
+* Fill in as much of the issue template as is relevant. Please be as specific as possible. Include the version of the code you were using, as well as what operating system you are running. If possible, include complete, minimal example code that reproduces the problem.
+
+## Setting up your development environment
+
+We recommend using the [conda](https://conda.io/docs/) package manager for your Python environments.
+
+Install [git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+(link with instructions) on your system if not already available
+(check with ``git --version`` at the command line.)
+This can also be installed from a variety of package managers, including ``conda`` if needed.
+
+Login to your [GitHub](https://github.com) account and make a fork of the
+[IOOS_QC repository](https://github.com/ioos/ioos_qc) by clicking the "Fork" button.
+Clone your fork of the repository (in terminal on Mac/Linux or git shell/GUI on Windows)
+to the location you'd like to keep it.
+We are partial to creating a ``git_repos`` or ``projects`` directory in our home folder.
+
+```sh
+git clone https://github.com/<your-user-name>/ioos_qc.git
+```
+
+Navigate to that folder in the terminal or in Anaconda Prompt if you're on Windows.
+The remainder of the instructions will take place within this directory.
+
+```sh
+cd ioos_qc
+```
+
+Connect your repository to the upstream (main project).
+
+```sh
+git remote add upstream https://github.com/ioos/ioos_qc.git
+```
+
+Create a new conda environment for us to configure, and give it a name.
+
+For Environment Setup:
+
+After ``-n`` you can specify any name you'd like; here we've chosen ``ioosqc38``.
```

Comment:
```suggestion
After ``-n`` you can specify any name you'd like; here we've chosen ``ioosqc312`.
```

Reply:
But in the documentation its, written the ioosqc38, so I guess we need to fix that as well.

Reply:
The contributing guide will replace any contributing instructions in the docs. We will remove those later in another PR. That is why I requested you to try to build it in a comment below.